diff --git a/absl/time/internal/cctz/BUILD.bazel b/absl/time/internal/cctz/BUILD.bazel
index 6e178748..aa9b463b 100644
--- a/absl/time/internal/cctz/BUILD.bazel
+++ b/absl/time/internal/cctz/BUILD.bazel
@@ -72,8 +72,8 @@ cc_library(
     ],
     copts = ABSL_DEFAULT_COPTS,
     linkopts = ABSL_DEFAULT_LINKOPTS + select({
-        "@platforms//os:osx": ["-Wl,-framework,CoreFoundation"],
-        "@platforms//os:ios": ["-Wl,-framework,CoreFoundation"],
+#        "@platforms//os:osx": ["-Wl,-framework,CoreFoundation"],
+#        "@platforms//os:ios": ["-Wl,-framework,CoreFoundation"],
         "//conditions:default": [],
     }),
     visibility = ["//visibility:public"],
